homeassistant/components/unifi/diagnostics.py

```diff
@@ -0,0 +1,29 @@
+"""Diagnostics support for UniFi Network."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN as UNIFI_DOMAIN
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant, config_entry: ConfigEntry
+) -> dict[str, Any]:
+    """Return diagnostics for a config entry."""
+    controller = hass.data[UNIFI_DOMAIN][config_entry.entry_id]
+    diag: dict[str, Any] = {}
+
+    diag["config_entry"] = dict(config_entry.data)
```

Comment:
Please redact sensitiv data from the output.
There is a helper fort that -> [`homeassistant.components.diagnostics.async_redact_data()`](https://github.com/home-assistant/core/blob/511546631a5b240c5ef4c0dd25c4a542edc62a72/homeassistant/components/diagnostics/util.py#L13)

Reply:
Redacted a lot of different stuff available in diagnostics